bundles/io.openliberty.tools.eclipse.product/about.properties

```diff
@@ -0,0 +1,6 @@
+blurb=Liberty Tools for Eclipse IDE\n\
+\n\
+Version: {featureVersion}\n\
```

Comment:
No issue...I'm just curious how you found this built-in property 'featureVersion'?   In the doc or just from some other project?

Reply:
I found it in some example Eclipse projects.